changes/ocd-4686.sql

```diff
@@ -0,0 +1,51 @@
+insert into openchpl.report_metadata (environment, title, report_key, report_group, url, height, display_order, last_modified_user)
+select 'DEV', 
+        'Service Base URL Listing Report',
```

Comment:
Is this name right? I thought the term was "Service Base URL List" but this says "Listing". Is it something different?